README.md

```diff
@@ -4,3 +4,11 @@ This is a placeholder file
 
 How we use version control:
 https://team2537.github.io/training/development-practices/version-control.html
+
+## Warning!
```

Comment:
Do we need to keep these warnings in the README if all the vendordeps listed here are now available?